README.md

```diff
@@ -2,30 +2,48 @@
 
 This is our working manifesto, which we keep as a public document. It has details specific to the product, engineering, and data teams, but a lot of it is just about how to work together effectively and bring out the best in each other.
```

Comment:
For readability, this might benefit from a Table of Contents

Reply:
So true, Jen ! Adding!! These suggestions are gold!

Reply:
okay, so I'm not sure how this TOC will render 😅. I created a gist to double check my work: https://gist.github.com/mirie/0fbaa915887c3cc5578a70813fc88e7e#data-collection

Reply:
love it!